Rhino.Etl.Core/Operations/AbstractDatabaseOperation.cs

```diff
@@ -49,6 +50,8 @@ protected AbstractDatabaseOperation(string connectionStringName)
                                              "Cannot resolve connection strings with name: " + connectionStringName);
 
             connectionStringSettings = ConfigurationManager.ConnectionStrings[connectionStringName];
+            if (connectionStringSettings != null && connectionStringSettings.ProviderName.Contains("Oracle"))
```

Comment:
Can we move this to a single location, so there is no repetition.
Better also if we would have `ParameterPrefix` that would be filled once